internal/cloudapi/v2/server.go

```diff
@@ -294,7 +294,7 @@ func (s *Server) enqueueKojiCompose(taskID uint64, server, name, version, releas
 		arch := ir.imageType.Arch()
 		distribution := arch.Distro()
 
-		manifestSource, _, err := ir.imageType.Manifest(&ibp, ir.imageOptions, ir.repositories, ir.manifestSeed)
+		manifestSource, _, err := ir.imageType.Manifest(&ibp, ir.imageOptions, ir.repositories, &ir.manifestSeed)
```

Comment:
You'll need to use the indexed version (`&irs[idx].manifestSeed`) to resolve the memory aliasing error.

Reply:
Thank you! Sorry for this extra round, fixed now.